### PR TITLE
Stats endpoint

### LINKED ISSUE #51: add basic uptime statistics
Need to add GET /stats end point to the bio auth service.  

The result returned should be json:
 {
            "serviceName": "guardian-bio-auth",
            "startedAt": "Tue Apr 19 2022",
            "currentTime": "Tue Apr 19 2022",
            "versions": [ "none" ]
}

For now, it is not important to add the versioning data (just return none as an element in the array so that we remain consistent with all of our other services).  We will have a second milestone to cover versioning data at a later date.

Please see this as a reference: 
https://github.com/kiva/protocol-reporting/blob/634f846789a43564801914361446cab625a2cb12/src/app/app.controller.ts#L37
https://github.com/kiva/protocol-reporting/blob/main/src/app/app.service.ts
https://docs.google.com/document/d/1AO1VcBiXKvFm_xPUenk0MEN6J1rowP0uFZt0wwxyxJ0/edit#heading=h.vfml1xnyi3gq

### PR DESCRIPTION
**Pull request by jeffk-kiva**
Resolves #51 

Adds basic stats endpoints to Bio Auth Service and Bioanalyzer Service.

### CODE REVIEW

File: bio_auth_service/src/main/kotlin/fingerprint/FingerprintRoutes.kt [hunk 1]
```diff
@@ -20,7 +19,6 @@ import org.kiva.bioauthservice.fingerprint.dtos.VerifyRequestDto
  */
 private object Paths {
     const val apiV1 = "/api/v1"
-    const val getPositions = "/positions/template"
```
Comment (natesulat) [inline bio_auth_service/src/main/kotlin/fingerprint/FingerprintRoutes.kt:23]: Expected?

**Review by natesulat — COMMENTED** [review]
Looks fine to me. Just want clarification on the one comment, but otherwise you've got my
giphy (5)

File: bio_auth_service/src/main/kotlin/fingerprint/FingerprintRoutes.kt (hunk 1 above)
Comment (jeffk-kiva) [reply bio_auth_service/src/main/kotlin/fingerprint/FingerprintRoutes.kt:23]: Yeah, I noticed it was an unused variable (and there was an unused import above too). I should probably tweak the lint settings to catch these things.

**Review by matt-raffel-kiva — APPROVED** [review]
lgtm, sans @natesulat comment

**Review by natesulat — APPROVED** [review]

**Pull request merged**